credsweeper/rules/config.yaml

```diff
@@ -97,7 +97,7 @@
   severity: high
   type: pattern
   values:
-    - (?P<value>EAACEdEose0cBA[0-9A-Za-z]+)
+    - (^|[^0-9A-Za-z])(?P<value>EAACEdEose0cBA[0-9A-Za-z]+)
```

Comment:
Isn't it need to be changed the right value?
I just wonder how the right value is added or not.

Reply:
The key pattern has no concrete size - rigth barier is extra.

Reply:
Ah, I see :)